modules/javafx.web/src/main/native/Source/cmake/OptionsMSVC.cmake

```diff
@@ -117,8 +117,15 @@ if (NOT COMPILER_IS_CLANG_CL)
     )
 endif ()
 
-# Create pdb files for debugging purposes, also for Release builds
-add_compile_options(/Zi /GS)
+if (PORT STREQUAL "Java")
+    # Suppress creation of pdb files for Release builds
+    # FIXME: Need to re-enable the flag for Debug builds
+    #add_compile_options(/Zi /GS)
```

Comment:
I can confirm that this restores the build logic we had prior to the WebKit 619.1 update, by removing the `/Zi` and `/GS` compile flags. See the following from the commit right before the last WebKit update:

https://github.com/openjdk/jfx/blob/e3c15957488256ec53c5fb9978e336c59b69d65e/modules/javafx.web/src/main/native/Source/cmake/OptionsMSVC.cmake#L119-L122

I also note that previously we had also removed the "`/DEBUG` linker flag, but since that isn't relevant to the errors we are seeing on some of our build machines, we could consider filing a  P4 follow-up bug to consider also removing the `/DEBUG` linker flag for our builds.